src/main/java/Task6/DayOfWeek.java

```diff
@@ -0,0 +1,19 @@
+package Task6;
+
+public class DayOfWeek {
+
+
+    public static String DaysOfWeek (int arg) {
+        String Day;
+        switch (arg) {
+            case 1:  Day= "Monday"; break;
+            case 2:  Day= "Tuesday"; break;
+            case 3:  Day= "Wednesday"; break;
```

Comment:
można to delikatnie uprościć i od razu zwrócić wartość zamiast przypisywać do zmiennej.

Reply:
Czyli użyć "System.out.println("Monday") itd?
Tylko wtedy metode zmieniam ze string na void, próbowałem void ale nie potrafiłem zrobić do niego testu. Chyba, że w jakiś inny sposób wywołać, ale nie udało mi się go znaleźć.

Reply:
Nie wywołać tylko zwrócić wartość zamiast przypisywać*